docs/install_guides/troubleshooting-steps.md

```diff
@@ -34,8 +34,6 @@ python -m pip install --upgrade pip
 
 ## Check if required external dependencies are installed
 
```

Comment:
```suggestion
For OpenVINO releases prior to 2022.2, if you are using Ubuntu or RHEL 8 systems, and installed OpenVINO Runtime via the installer, APT, or YUM repository, and then decided to [install OpenVINO Development Tools](installing-model-dev-tools.md), make sure that you **Install External Software Dependencies** first by following the steps in the corresponding installation pages.
```
Upon second thought, Troubleshooting might be a little different so we need keep this somehow.